src/main/java/org/kitteh/irc/client/library/command/KickCommand.java

```diff
@@ -43,6 +43,19 @@
     /**
      * Constructs a KICK command for a given channel.
      *
+     * @param channel channel targeted
+     * @throws IllegalArgumentException if null parameters or Channel is from
+     * another Client
+     * @see Channel#kick(User)
+     * @see Channel#kick(User, String)
+     */
+    public KickCommand(@Nonnull Channel channel) {
```

Comment:
Does this use their name/void as reason? Is there a default kick reason in KICL? would be useful in the javadocs to know what the kick reason will be